modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinMemoryLeakTest.java

```diff
@@ -168,7 +168,7 @@ public static Collection<Object[]> data() {
                 DatePicker.class,
                 MenuBar.class,
                 MenuButton.class,
-                Pagination.class,
+                //Pagination.class,
```

Comment:
I would just remove this (which was done for other classes once the leaks were fixed).

Reply:
Keeping this to avoid merge conflicts.  Will remove the whole exclusion block when the last skin is fixed.